android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/ssdp/SsdpMessage.java

```diff
@@ -0,0 +1,90 @@
+
+
+package org.physical_web.physicalweb.ssdp;
```

Comment:
Please move this line below the licensing text in order to match the other classes.